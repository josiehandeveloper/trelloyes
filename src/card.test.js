import React from 'react';
import ReactDOM from 'react-dom';
import Card from './card';
import renderer from 'react-test-renderer';

describe('Card component', () => {

    //Smoke test
    it('renders without crashing', () => {
        const div = document.createElement('div');
        ReactDOM.render(<Card />, div);
        ReactDOM.unmountComponentAtNode(div);
    });
    
    it('renders the UI as expected', () => {
         const tree = renderer
        .create(<Card title="title" content="content"/>)
        .toJSON();
        expect(tree).toMatchSnapshot();  
    });
});

