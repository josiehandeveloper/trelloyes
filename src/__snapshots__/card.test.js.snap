// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Card component renders the UI as expected 1`] = `
<div
  className="card"
>
  <button
    type="button"
  >
    delete
  </button>
  <h3>
    title
  </h3>
  <p>
    content
  </p>
</div>
`;

exports[`renders the UI as expected 1`] = `
<div
  className="card"
>
  <button
    type="button"
  >
    delete
  </button>
  <h3 />
  <p />
</div>
`;
